src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components/macro';
+import home from './Home.svg';
```

Comment:
Since this is a `svg` you could directly import it as an `svg` and not add it as an `img`:
```suggestion
import {ReactComponent as Home} from './Home.svg';
```
You can then instead of the `image` add the svg component:
```
<Home />
```